monkestation/code/modules/ghost_players/arena/fight_button.dm

```diff
@@ -134,41 +134,37 @@
 		say("One or more of the players have left the area, match has been cancelled!")
 		return
 
-
 	if(!player_one.client.prefs.adjust_metacoins(player_one.ckey, -payout, "Added to the Payout"))
 		return
-	if(!player_two.client.prefs.adjust_metacoins(player_one.ckey, -payout, "Added to the Payout"))
+	if(!player_two.client.prefs.adjust_metacoins(player_two.ckey, -payout, "Added to the Payout"))
 		player_one.client.prefs.adjust_metacoins(player_one.ckey, payout, "Opponent left, reimbursed.")
 		return
 
+	var/turf/player_one_spot = locate(148, 34, SSmapping.levels_by_trait(ZTRAIT_CENTCOM)[1])
+	prep_player(player_one, player_one_spot)
+	var/turf/player_two_spot = locate(164, 34, SSmapping.levels_by_trait(ZTRAIT_CENTCOM)[1])
```

Comment:
these start spots are hardcoded, gross. Ideally these should be landmarks instead

Reply:
While I agree, I feel that should be done in a separate PR. These were hardcoded before, and to de-hardcode them, I'd have to make changes to the centcom map - which means conflicting with any other PRs that touch the centcom map.

Reply:
Fair. It should absolutely be changed in the future, though